src/Middleware/passingNodeData.ts

```diff
@@ -21,6 +21,9 @@ export async function passNodeData(
                     res.locals.node = node;
                 }
             }
+        } else {
+            const miningRewards = req.body;
+            res.locals.miningRewards = miningRewards;
```

Comment:
Middleware will pass the body to the controller, no need to define it in the locals